src/bp/ui-shared/src/index.tsx

```diff
@@ -49,3 +50,7 @@ exports.lang = {
 }
 exports.toast = toast
 exports.utils = { controlKey, keyMap, isInputFocused }
+exports.telemetry = {
+  startOfflineTelemetryFallback,
+  sendEventsToTelemetryServer
```

Comment:
You are already in a "namespace" for telemetry, please keep the method name short: sendEvents, startWatcher for example.

Reply:
Fixed in 400ec94